pkgs/games/abuse/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, makeDesktopItem, copyDesktopItems, SDL, SDL_mixer, freepats }:
+
+stdenv.mkDerivation rec {
+  pname   = "abuse";
+  version = "0.8";
+
+  src = fetchurl {
+    url    = "http://abuse.zoy.org/raw-attachment/wiki/download/${pname}-${version}.tar.gz";
+    sha256 = "0104db5fd2695c9518583783f7aaa7e5c0355e27c5a803840a05aef97f9d3488";
+  };
+
+  configureFlags = [
+    "--with-x"
+    "--with-assetdir=$(out)/orig"
```

Comment:
```suggestion
    "--with-assetdir=$out/orig"
```

Reply:
Why so? It happens to work without the parentheses, but I guess this is not something to rely on. This string will be interpolated into the Makefile, where the syntax to access environment variables is with the parentheses. (Without the parentheses, gcc is invoked with `-DASSETDIR="ut/orig"`.)